### PR TITLE
rpc: implement getstorage RPC

### LINKED ISSUE #343: RPC: implement getstorage

### PR DESCRIPTION
**Pull request by fyrchik**
Closes #343 .

### CODE REVIEW

File: pkg/rpc/server.go [hunk 1]
```diff
@@ -207,6 +207,9 @@ Methods:
 
 		results = peers
 
+	case "getstorage":
+		results, resultsErr = s.getStorage(reqParams)
```
Comment (roman-khimov) [inline pkg/rpc/server.go]: Missing Prometheus counter for this call.

File: pkg/rpc/server.go [hunk 2]
```diff
@@ -274,6 +277,31 @@ Methods:
...
+		return nil, errInvalidParams
+	}
+
+	key, err := param.GetBytesHex()
+	if err != nil {
+		return nil, errInvalidParams
+	}
+
+	item := s.chain.GetStorageItem(scriptHash.Reverse(), key)
+	return item, nil
```
Comment (roman-khimov) [inline pkg/rpc/server.go]: Is it JSONized correctly or are we missing something here?

**Comment by roman-khimov** [comment]
Also, documentation should be updated.

File: pkg/rpc/server.go (hunk 2 above)
Comment (fyrchik) [reply pkg/rpc/server.go]: Yes, we are. Fixed.
Query:
```json
{
    "jsonrpc": "2.0",
    "method": "getstorage",
    "params": [
        "67a5086bac196b67d5fd20745b0dc9db4d2930ed",
        "8eb60cf0dd78bb881d8bc1ccdc24e3c4aa5576de"
    ],
    "id": 1
}
```
Result:
```json
{
    "jsonrpc": "2.0",
    "id": 1,
    "result": "002e597611"
}
```

**Comment by roman-khimov** [comment]
Care to add some tests for it?

**Comment by fyrchik** [comment]
@roman-khimov The easiest way to do it (just put in storage and test RPC) requires exporting `makeStorageItemKey` or `newDao` which is not the right way to do it IMO.
I will postpone it until #675 will be merged and there will be some storage in test blocks already.

**Comment by fyrchik** [comment]
Updated PR with new testchain and tests.

**Pull request merged**